inspirehep/modules/workflows/tasks/actions.py

```diff
@@ -111,3 +113,10 @@ def is_record_relevant(obj, *args, **kwargs):
                 len(core_keywords) == 0:
             return False
     return True
+
+
+def is_experimental_paper(obj, eng):
+    """Is the record an experimental paper."""
+    categories = get_value(obj.data, "arxiv_eprints.categories", []) + \
+        get_value(obj.data, "subject_terms.term", [])
+    return "hep-ex" in categories or "Experiment-HEP" in categories
```

Comment:
Yeah, indeed there are probably better criterias. We could look in the collaboration field as well as your inspire search indicate - however do we need to guess the experiment at all if collaboration found to be ATLAS? To rephrase, could we rely on assigning experiments via the collaboration field when available as well?